### DISCUSSION TITLE
How is mercury compared to quarto

### DISCUSSION THREAD

**Discussion by abdelkareemkobo** [mljar/mercury#353] [General]
I was using quarto for a while to create an interactive blogs from jupyter notebooks!
But i want to embed some interactive plots into my blogs is it good with mercury or Will give me some issues related to management and performance

**Comment by pplonski** [comment]
Quarto is for static documents. Mercury is for interactive web apps.

It all depends what do you mean by interactive plots? If you just want to provide zoom in/zoom then you can go with Quarto. If you would like to interact with your data source with widgets then you can go with Mercury.
